### PR TITLE
Update readme to reflect new user env settings

### PR DESCRIPTION
**Pull request by Timelapser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,40 +97,12 @@ variables `LLDAP_JWT_SECRET_FILE` or `LLDAP_LDAP_USER_PASS_FILE`, and the file
...
-      - "lldap_data:/data"
-      # Alternatively, you can mount a local folder
-      # - "./lldap_data:/data"
-    environment:
-      - LLDAP_JWT_SECRET=REPLACE_WITH_RANDOM
-      - LLDAP_LDAP_USER_PASS=REPLACE_WITH_PASSWORD
-      - LLDAP_LDAP_BASE_DN=dc=example,dc=com
-```
-
 Example for docker compose for `:latest` tag:
```
Comment (nitnelave) [inline README.md:100]: I think we can just say "example for docker compose" here

File: README.md [hunk 2]
```diff
@@ -97,40 +97,12 @@ variables `LLDAP_JWT_SECRET_FILE` or `LLDAP_LDAP_USER_PASS_FILE`, and the file
...
-* `:latest` tag image contain recent pushed codes or feature test, breaks is expected.
+* `:latest` tag image contains recently pushed code or feature tests, some broken parts are to be expected.
```
Comment (nitnelave) [inline README.md:101]: Maybe "some instability is to be expected"?

**Review by nitnelave — APPROVED** [review]
Overall good, just a couple of nits

**Comment by Timelapser** [comment]
Updated it & also changed the latest tag to stable in the example code!

**Review by nitnelave — APPROVED** [review]

**Comment by nitnelave** [comment]
Thanks!

**Pull request merged**